src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -1400,13 +1400,15 @@ std::string TinyGltfImporter::doMaterialName(const UnsignedInt id) {
 /* textureMatrix should be an empty Optional when parsing the first texture.
    The function will fill it and then use to check consistency of the transform
    for subsequent textures. */
-bool TinyGltfImporter::materialTexture(const char* name, const Int texture, const Int texCoord, const tinygltf::Value& extensions, UnsignedInt& index, Containers::Optional<Matrix3>& textureMatrix, PhongMaterialData::Flags& flags) const {
+bool TinyGltfImporter::materialTexture(const char* name, const Int texture, const Int texCoord, const tinygltf::Value& extensions, UnsignedInt& index, UnsignedInt& coordinateSet, Containers::Optional<Matrix3>& textureMatrix, PhongMaterialData::Flags& flags) const {
     if(UnsignedInt(texture) >= _d->model.textures.size()) {
         Error{} << "Trade::TinyGltfImporter::material():" << name << "index" << texture << "out of bounds for" << _d->model.textures.size() << "textures";
         return false;
     }
 
-    if(texCoord != 0) {
+    if(configuration().value<bool>("allowMaterialTextureCoordinateSets")) {
+        coordinateSet = texCoord;
+    } else if(texCoord != 0) {
         Error{} << "Trade::TinyGltfImporter::material(): multiple texture coordinate sets are not supported";
         return false;
     }
```

Comment:
```suggestion
    if(texCoord != 0) {
        if(!configuration().value<bool>("allowMaterialTextureCoordinateSets")) {
            Error{} << "Trade::TinyGltfImporter::material(): multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them";
            return false;
        }
        
        coordinateSet = texCoord;
    }
```

Basically the same as in the case of Assimp.